user_guide/intro_navigation.rst

```diff
@@ -0,0 +1,110 @@
+Navigation Basics
+=================
+
+OroCRM provides a broad set of navigation tools that allow you to easily access necessary information and tools. The 
+system helps to ensure that the frequent actions can be performed with minimal effort by keeping required information at 
+hand. This means no important updates are missed.
+
+This section provides an overview of the navigation elements in OroCRM.
+
+
+.. _user-guide-navigation-menu:
+
+The Menu Bar
+------------
+
+The menu bar is the main mode of accessing data in OroCRM. It is always displayed at every page of OroCRM. Subject to 
+your organization configuration, it may be displayed horizontally or vertically. In the latter case, the menu items will 
+be displayed as icons.
+
+|
+
+.. image:: ./img/navigation/menu/nav_bar_top.png
+
+|
+
+.. image:: ./img/navigation/menu/nav_bar_side.png
+
+|
+
+In the mobile version the menu bar can be accessed via the "hamburger" button next to the organization name:
+
+|
+    
+.. image:: ./img/navigation/menu/header_mobile.png
+
+|
+
+The list of menu items available to you, depends on the access and permission settings defined in the organization.
+
+Hover the mouse over a menu item to expand the drop-down menu with the second-level items. Click on them to access 
+corresponding pages or grids. (See :ref:`Data management basics <user-guide-data-management-basics>` for further 
+details.)
+  
+  
+.. _user-guide-navigation-panel:
+
+Navigation Tools
+----------------
+
+At the very top of the page, there is navigation panel that contains different navigation tools, as shown below.
+
+|
+
+.. image:: ./img/navigation/panel.png
+
+
+.. csv-table::
+  :header: "#","Item","Description","Related Document"
+  :widths: 10, 30, 40,30
+  
+  "1","**Organization Selector**","Switch between organizations when working in  the multi-organization environment.","
+  :ref:`Working with Multiple Organizations <user-guide-getting-started-change-organization>` section of the 
+  *Company Structure and Organization Selector* guide`"
+  "2","**Shortcuts**",Quickly perform the most common actions from anywhere in the system.","
+  :ref:`Shortcuts <user-guide-getting-started-shortcuts>`"
+  "3","**Search**","Find a specific record's 
+  :ref:`View page <user-guide-ui-components-view-pages>`","
+  :ref:`Search <user-guide-getting-started-search>`"
+  "4","**Pinned pages**","Open pages preserved in the state they were pinned in.",
+  ":ref:`Pinned Page <user-guide-ui-components-pinned>`"
+  "5","**User menu**","Access your user profile, calendar, mailbox, and task list. Log out of the system.","
+  :ref:`User Menu <user-guide-getting-started-user-menu>`"
+  "6","**My Emails**","See if you have new and unread emails. Quickly read and reply to most recent emails. 
+  
+  The envelop icon reflect current state of your mailbox - from 
+  the menu under it, you can get to the *My Emails* page, read and reply any of the most recent 5 emails and mark all 
+  the emails as read",""
```

Comment:
Edit: The envelop icon reflects the current state of your mailbox - from the menu under it, you can get to the *My Emails* page, read and reply any of the most recent five emails, and mark all the emails as read",""